chingu-coders/Voyage2-Turtles-02
# css validator fix, created components dir

## Purpose
* To get a bare-minimum functional CSS validator
* to begin the process of reorganizing the `js` directory structure

## Approach/Fixes
* CSS validator needed a proxy to make CORS request. I added one.
* As for the directory structure, the JS is starting to really pile up, so we decided to start separating `main.js` into separate files. @ziggysauce had suggested one file for each contributor, but after thinking about it, that doesn't make sense semantically, since the code has nothing to do with who wrote it. After researching, it seems to be standard practice to create a `components` directory and separate the JS out into separate component files. Thats what I did with my code. If y'all arent feeling it, we can scrap it and go the other direction.

## Testing
* Checkout this pull request.
* open the CSS validator and input some invalid CSS.
* confirm that error `type` `line` and `message` are displayed
* note that the appearance needs A LOT of work, but the validator is functional
